.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,13 @@
-https://github.com/nimble/git-template/issues/??
+https://github.com/nimblehq/git-template/issues/??
```

Comment:
Should we modify this link?
```suggestion
https://github.com/nimblehq/infrastructure-templates/issues/??
```

Reply:
thanks @hoangmirs , adjusted in https://github.com/nimblehq/infrastructure-templates/pull/21/commits/89391c8c137cf3058b545bfe6854f6e950a747cb